packages/cardano-services/src/Program/programs/projector.ts

```diff
@@ -61,6 +63,7 @@ const createProjectionHttpService = async (options: ProjectionMapFactoryOptions)
       handlePolicyIds
     },
     projectionSource$: Bootstrap.fromCardanoNode({
+      blocksBufferLength,
```

Comment:
Question: are we sure this is mapped correctly from the cli arg/env?

Reply:
Actually we do not: I haven't tested this, nice catch!
The only idea I have is to log it from the projector and check the logs in `cli.test.ts`.
Does it sounds as a good test or do you have some better idea?
Thank you

Reply:
Yeah that would be fine, it's probably really difficult to set up a proper test, need to abstract away the cli :sweat_smile:

Reply:
1d07bf077 should solve this